internal/api/ring.go

```diff
@@ -342,8 +343,10 @@ func handleReleaseAllRings(c *Context, w http.ResponseWriter, r *http.Request) {
 			if ring.Image != releaseRingRequest.Image || ring.Version != releaseRingRequest.Version {
 
 				ring.State = model.RingStateReleasePending
-				ring.Image = releaseRingRequest.Image
-				ring.Version = releaseRingRequest.Version
+				ring.ChangeRequest.Image = releaseRingRequest.Image
+				ring.ChangeRequest.Version = releaseRingRequest.Version
+				ring.ChangeRequest.ReleaseStart = time.Now().UnixNano()
```

Comment:
I think in most places we are using milliseconds, so we should stick to it unless there is very good reason not to. Because we store time as `int64` and not `time.Time` this can lead to hard to find bugs if we start comparing millis to nanos.

Reply:
We actually use Nano everywhere in Elrond, so unless there is a benefit I am missing I believe we can stay with that right?